### PR TITLE
chore(test): Update to Firefox 67 and intern@4.4.3

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Let's see if this works!

### CODE REVIEW

File: packages/fxa-content-server/package.json [hunk 1]
```diff
@@ -155,7 +155,7 @@
         "grunt-todo": "0.5.0",
         "htmlparser2": "3.9.0",
         "install": "0.12.1",
-        "intern": "4.3.1",
+        "intern": "^4.4.3",
         "jsqr": "1.2.0",
         "leadfoot": "1.7.4",
```
Comment (vladikoff) [inline packages/fxa-content-server/package.json]: might need to bump this too here

**Comment by chenba** [comment]
🤞

File: packages/fxa-content-server/package.json (hunk 1 above)
Comment (chenba) [reply packages/fxa-content-server/package.json]: That is the latest leadfoot on npm.  `intern` itself uses `@theintern/leadfoot`, which is much newer.

Comment (vladikoff) [reply packages/fxa-content-server/package.json]: should switch to `@theintern/leadfoot` here then

File: .circleci/test-content-server.sh [hunk 2]
```diff
@@ -49,7 +49,7 @@ if grep -e "$MODULE" -e 'all' $DIR/../packages/test.list; then
   # ensure email-service is ready
   check 127.0.0.1:8001/__heartbeat__
   cd packages/fxa-content-server
-  mozdownload --version 58.0 --destination firefox.tar.bz2
+  mozdownload --version 67.0 --destination firefox.tar.bz2
```
Comment (6a68) [inline .circleci/test-content-server.sh]: @shane-tomlinson Is there some reason we wouldn't use `latest` here, instead of a specific version number?

Comment (shane-tomlinson) [reply .circleci/test-content-server.sh]: The biggest reason is that sometimes digdug/theintern do not work with the latest and greatest.

**Pull request merged**